client/my-sites/earn/home.tsx

```diff
@@ -504,19 +508,24 @@ export default connect< ConnectedProps, {}, {} >(
 	( state ) => {
 		const selectedSiteSlug = getSelectedSiteSlug( state );
 		const site = getSiteBySlug( state, selectedSiteSlug );
+		const isFreePlan = ! isCurrentPlanPaid( state, site.ID );
+		const hasConnectedAccount = get(
+			state,
+			[ 'memberships', 'settings', site.ID, 'connectedAccountId' ],
+			null
+		);
```

Comment:
Non-blocking" We're using the babel preset-env in our build, so we can use nullish coalescing and optional chaining instead of lodash get. Usage would look something like:

```jsx
const hasConnectionAccount = state?.memberships?.settings?.[ siteId ]?.connectedAccountId ?? null; //note that we can add a follow up janitorial to create a selector function instead
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Oh, nice. Noted for future reference.